### PR TITLE
Sphinxyphying social coding lesson closes #58

### LINKED ISSUE #58: Integrate slides into sphinx lesson material
When working through the material I noticed that there is a lot of slides, texts and information for the 75 min including break time in the workshop.
So this is a suggestion to have full text material as 'normal sphinx lesson' and then a shorter slideset to present with less text, see also here: https://coderefinery.zulipchat.com/#narrow/stream/136238-lessons/topic/social.20coding.20lesson
You can see a basic draft of how the lesson could look like here : https://github.com/samumantha/sc_test/tree/sphinxiphy
This is only the content of slides transformed into sphinx lesson with the sphinx lesson template.
For having it as material, some more adjustments will need to be made. But first I wanted to ask your opinion on this (making the initial draft was good practice and not much work, so no worries if you think this is not needed). 

So, what do you think, should I continue with this?

### PR DESCRIPTION
**Pull request by samumantha**
content of talk.md moved into sphinx lesson template.

Draft until all keypoints, other boxes etc are filled.

Exercise suggestions as usual very welcome.

Please review and add comments and issues on things that should be updated better :)

### CODE REVIEW

**Comment by annefou** [comment]
Great! Maybe we should already merge this PR and add changes on tp of it (rather than keeping it as draft). What do you think?

**Comment by samumantha** [comment]
Sure, maybe lets adjust the keypoints of licensing and social coding first, so that it is complete?
So, what should be the keypoints there?
Also, please check the objectives of all three lessons. I just put something.
After that we can merge and then continue to work on it?

**Comment by annefou** [comment]
Ok. I was thinking to adjust everything after we merge.

File: content/conf.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "LESSON NAME"
```
Comment (annefou) [inline content/conf.py:20]: Maybe we could already put the lesson name "Social Coding"

File: content/software_citation.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Who owns the code that you write?
+- Does your code/script have a license?
+- Can you take your own code with you after the PhD?
+- Are you using open source software in your work?
+- What are your thoughts about sharing software with your colleagues or competitors?
+```
+
+```{keypoints}
+- License your code **very early** in the project:
+  ideally develop publicly accessible open source code **from day one**.
```
Comment (annefou) [inline content/software_citation.md:59]: Is it a key point here or in the previous episode?

I would add another key point:
- do not forget to cite codes you use

**Review by annefou — APPROVED** [review]
I think this is really great. I really like the new template!

**Comment by samumantha** [comment]
Yeah, I also really like the template! 
Shall we then continue updating with assigned issues?

**Comment by samumantha** [comment]
closes #59

**Pull request merged**